konig-spreadsheet/src/test/java/io/konig/spreadsheet/WorkbookLoaderTest.java

```diff
@@ -1091,6 +1091,66 @@ public void testSecurityClassification() throws Exception {
 		assertEquals(DCL4, sc.iterator().next());
 
 	}
+	@Test
+	public void testTabularNodeShape_1() throws Exception {
+
+		InputStream input = getClass().getClassLoader().getResourceAsStream("rdbms-node-shape.xlsx");
+		Workbook book = WorkbookFactory.create(input);
+		Graph graph = new MemoryGraph();
+		NamespaceManager nsManager = new MemoryNamespaceManager();
+		graph.setNamespaceManager(nsManager);
+		
+		WorkbookLoader loader = new WorkbookLoader(nsManager);
+		loader.load(book, graph);
+		input.close();
+		
+		URI shapeId = uri("http://example.com/shapes/SourcePersonShape");
+		
+		ShapeManager s = new MemoryShapeManager();
+		
+		ShapeLoader shapeLoader = new ShapeLoader(s);
+		shapeLoader.load(graph);
+		
+		ShapeWriter shapeWriter = new ShapeWriter();
+		Shape shape = s.getShapeById(shapeId);
+		
+		assertTrue(shape!=null);
```

Comment:
You need to add more assertions.  What exactly is this test supposed to be validating?